rules/falco_rules.yaml

```diff
@@ -299,6 +299,9 @@
 - macro: container
   condition: container.id != host
 
+- macro: container_started
+  condition: (evt.type = container or (evt.type=execve and evt.dir=< and proc.vpid=1))
```

Comment:
Just want to drop a note here, if the container is brought up with flag `--pid=host` or `--pid=<container id or name>`, the vpid wont be equal to 1. We might miss the containers brought up with pid namespace configured. For more reference: https://docs.docker.com/engine/reference/run/#imagedigest

Reply:
Yeah container.id = host allows you to know that you're in a container. proc.vpid=1 was the best way we had to denote that a container was started. Now we'll have a more reliable way using the container event.